Assets/Scripts/UI/DialogBox/Options/DialogBoxOptions.cs

```diff
@@ -31,45 +27,83 @@ public void OnButtonLoadGame()
         dialogManager.dialogBoxLoadGame.ShowDialog();
     }
 
+    public void OnButtonOpenSettings()
+    {
+        this.CloseDialog();
+        dialogManager.dialogBoxSettings.ShowDialog();
+    }
+
     // Quit the app whether in editor or a build version.
     public void OnButtonQuitGame()
     {
         // Maybe ask the user if he want to save or is sure they want to quit??
-        #if UNITY_EDITOR
+#if UNITY_EDITOR
         // Allows you to quit in the editor.
         UnityEditor.EditorApplication.isPlaying = false;
-        #else
+#else
         Application.Quit();
-        #endif
+#endif
     }
 
-    private void Start()
+    private void RenderButtons()
     {
-        dialogManager = GameObject.FindObjectOfType<DialogBoxManager>();
-        WorldController wc = GameObject.FindObjectOfType<WorldController>();
+        Object buttonPrefab = Resources.Load("UI/Components/MenuButton");
 
-        // Add listeners here.
-        buttonQuit.onClick.AddListener(delegate
-        {
-            OnButtonQuitGame();
-        });
-        buttonResume.onClick.AddListener(delegate
+        GameObject resumeButton = CreateButtonGO(buttonPrefab, "Resume", "menu_resume");
+        resumeButton.GetComponent<Button>().onClick.AddListener(delegate
         {
             this.CloseDialog();
         });
-        buttonNewWorld.onClick.AddListener(delegate
+
+        GameObject newWorldButton = CreateButtonGO(buttonPrefab, "New World", "new_world");
+        newWorldButton.GetComponent<Button>().onClick.AddListener(delegate
         {
-            wc.NewWorld();
+            OnButtonSaveGame();
         });
 
-        buttonSave.onClick.AddListener(delegate
+        GameObject saveButton = CreateButtonGO(buttonPrefab, "Save", "save");
+        saveButton.GetComponent<Button>().onClick.AddListener(delegate
         {
             OnButtonSaveGame();
         });
-        buttonLoad.onClick.AddListener(delegate
+
+        GameObject loadButton = CreateButtonGO(buttonPrefab, "Load", "load");
+        loadButton.GetComponent<Button>().onClick.AddListener(delegate
         {
             OnButtonLoadGame();
         });
+
+        GameObject settingsButton = CreateButtonGO(buttonPrefab, "Settings", "menu_settings");
+        settingsButton.GetComponent<Button>().onClick.AddListener(delegate
+        {
+            OnButtonOpenSettings();
+        });
+
+        GameObject quitButton = CreateButtonGO(buttonPrefab, "Quit", "menu_quit");
+        quitButton.GetComponent<Button>().onClick.AddListener(delegate
+        {
+            OnButtonQuitGame();
+        });
+    }
+
+    private GameObject CreateButtonGO(Object buttonPrefab, string name, string localizationCode)
```

Comment:
The name will set a understandable tag in the hierarchy tree, localization code would look inconsistent.